meta-mender-tegra/recipes-mender/mender-client/files/mender

```diff
@@ -10,6 +10,11 @@ if [ $? -eq 0 ]; then
     # Exit with failure and error message if we don't have alignment
     # between boot slot and rootfs.  It's not safe to update in this case
     mender-tegra-verify-boot-rootfs-slot-alignment || exit 1
+    if [ -e /var/volatile/mender-tegra-boot-slot-mismatch-install-disabled ]; then
```

Comment:
Suggest using `/run` instead of `/var/volatile` as the location for this sentinel file.

Reply:
Thanks for the suggestion!  @jajoosiddhant can you please make this change before you test on Monday?

Reply:
Sure

Reply:
Verified